packages/amazonq/src/inlineChat/controller/inlineChatController.ts

```diff
@@ -191,7 +191,6 @@ export class InlineChatController {
             })
             .then(async (query) => {
                 if (!query || query.trim() === '') {
-                    void vscode.window.showWarningMessage('Amazon Q: Instructions for cannot be empty')
                     return
```

Comment:
should it be logged, or is this a situation that we are certain won't need troubleshooting?

Reply:
Added log statement 👍